src/utils/rest-api.js

```diff
@@ -469,21 +476,52 @@ function fetchEquipmentInfos(
     studyUuid,
     selectedNodeUuid,
     equipmentPath,
-    equipmentId
+    equipmentId,
+    searchInUpstreamBuiltParent
 ) {
     console.info(
         `Fetching specific equipments '${equipmentId}' of type '${equipmentPath}' of study '${studyUuid}' and node '${selectedNodeUuid}' ...`
     );
+
+    let urlSearchParams = new URLSearchParams();
+    if (searchInUpstreamBuiltParent !== undefined) {
+        urlSearchParams.append(
+            'searchInUpstreamBuiltParentNode',
+            searchInUpstreamBuiltParent
+        );
+    }
+
     const fetchEquipmentInfosUrl =
         getStudyUrlWithNodeUuid(studyUuid, selectedNodeUuid) +
         '/network-map/' +
         equipmentPath +
         '/' +
-        encodeURIComponent(equipmentId);
+        encodeURIComponent(equipmentId) +
+        '?' +
+        urlSearchParams.toString();
     console.debug(fetchEquipmentInfosUrl);
     return backendFetch(fetchEquipmentInfosUrl);
 }
 
+export function fetchEquipmentExists(
+    studyUuid,
+    selectedNodeUuid,
+    equipmentPath,
+    equipmentId
+) {
+    console.info(
+        `Fetching existence of specific equipments '${equipmentId}' of type '${equipmentPath}' of study '${studyUuid}' and node '${selectedNodeUuid}' ...`
+    );
+    const fetchEquipmentInfosUrl =
```

Comment:
rename `fetchEquipmentInfosUrl` to `fetchEquipmentExistsUrl` to avoid confusion

Reply:
done